src/components/ItaliaTheme/View/BandoView/BandoView.jsx

```diff
@@ -91,6 +95,9 @@ const BandoView = ({ content, location }) => {
   return (
     <>
       <div className="container px-4 my-4 bando-view">
+        <a className="sr-only" href="#main-content-section">
+          {intl.formatMessage(messages.content)}
```

Comment:
Dato che viene ripetutto 1000 volte, non possiamo fare un componente a cui gli si passa l'id? o è troppo?

Reply:
Approvo, lo metto in Commons